pkg/config/remote/service/service_test.go

```diff
@@ -406,6 +416,33 @@ func TestClientGetConfigsRequestMissingFields(t *testing.T) {
 	assert.Equal(t, status.Convert(err).Code(), codes.InvalidArgument)
 }
 
+func TestClientGetConfigsEmptiesCacheForExpiredSignature(t *testing.T) {
```

Comment:
Nit: Nothing about this actually asserts that this empties the cache, it's more so just detecting that we send the EmptyCache message. The name right now implies a little more confidence than we can get from the test.